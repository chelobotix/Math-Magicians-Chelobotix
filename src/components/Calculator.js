import './calculator.css';
import Display from './Display';
import Pad from './Pad';

const CalculatorContainer = () => (
  <main>
    <Display />
    <Pad />
  </main>
);

export default CalculatorContainer;
