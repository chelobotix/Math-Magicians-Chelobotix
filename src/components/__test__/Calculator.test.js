import renderer from 'react-test-renderer';
import Calculator from '../Calculator';

describe('Calculator Component', () => {
  test('Tree', () => {
    const tree = renderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
