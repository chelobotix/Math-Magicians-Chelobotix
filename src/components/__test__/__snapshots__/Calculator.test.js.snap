// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Calculator Component Tree 1`] = `
<section
  className="section-calculator"
>
  <h2
    className="h2"
  >
    Let\`s do some math!
  </h2>
  <div
    className="calc-container"
  >
    <div
      className="display"
    >
      <p
        className="display"
      >
        0
      </p>
    </div>
    <div
      className="symbols-container"
    >
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="symbol"
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</section>
`;
